src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                                        dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                                        dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ,
+                                                       <https://doi.org/10.1130/9780813774022> ;
+                                        skos:definition "The basic formal unit in magnetostratigraphic polarity classification is the Magnetostratigraphic Polarity Zone (or Polarity Zone). The corresponding interval of geological time during which the rocks in the Polarity Zone were deposited is referred to as its associated Polarity Chron. (Salvador, 2013; ICS, Online, December, 2019)"@en ;
+                                        skos:prefLabel "magnetostratigraphic polarity zone"@en .
+
+
+###  http://sweetontology.net/realmGeol/MagnetostratigraphicUnit
+soreag:MagnetostratigraphicUnit rdf:type owl:Class ;
+                                rdfs:subClassOf soreag:StratigraphicUnit ;
+                                dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                                dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ,
+                                               <https://doi.org/10.1130/9780813774022> ;
+                                skos:definition "A Magnetostratigraphic Unit (Magnetozone) is a body of rocks unified by similar magnetic characteristics which allow it to be differentiated from adjacent rock bodies.  The term is collectively applied to the different kinds of stratigraphic units that are identified by their measurable magnetic properties. (Salvador, 2013; ICS, Online, December, 2019)"@en ;
+                                skos:prefLabel "magnetostratigraphic unit"@en .
 
 
 ###  http://sweetontology.net/realmGeol/Mantle
 soreag:Mantle rdf:type owl:Class ;
-            rdfs:subClassOf soreag:GeosphereLayer ,
-                            [ rdf:type owl:Restriction ;
-                              owl:onProperty sorelsp:hasUpperBoundary ;
-                              owl:allValuesFrom soreag:Moho
-                            ] ;
-            rdfs:label "mantle"@en .
+              rdfs:subClassOf soreag:GeosphereLayer ,
+                              [ rdf:type owl:Restriction ;
+                                owl:onProperty sorelsp:hasUpperBoundary ;
+                                owl:allValuesFrom soreag:Moho
+                              ] ;
+              rdfs:label "mantle"@en .
+
+
+###  http://sweetontology.net/realmGeol/Member
+soreag:Member rdf:type owl:Class ;
+              rdfs:subClassOf soreag:LithostratigraphicUnit ;
+              dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+              dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ;
+              skos:definition "The formal lithostratigraphic unit next in rank below a formation. (ICS, Online, December 2019)"@en ;
```

Comment:
A formal geologic unit (link to def) that is a proper part of a formation (link to def)